test/src/api/commands/web-element/testIsVisible.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isVisible() command', function() {
+  before(function(done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function(done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isVisible() Visible', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/Visible',
+      method: 'GET',
+      response: JSON.stringify({
+        value: true
+      })
+    }, true);
+
+    this.client.api.element('#search').isVisible(function(result) {
+      this.assert.equal(result.value, true);
+    });
```

Comment:
`isVisible()` command does not accept a callback due to which the assertion will never run and the test is basically useless.

As I've said many times before, please refer to the `test/src/api/commands/web-element/testGetText.js` file while writing these tests.